static/js/ContentText.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Sefaria from "./sefaria/sefaria";
+
+const ContentText = (props) => {
+  /* Renders content language throughout the site (content that comes from the database and is not interface language).
+  `html` is present
+  * See filterContentTextByLang for documentation */
+  const langAndContentItems = filterContentTextByLang(props);
+  return langAndContentItems.map(item => <ContentSpan lang={item[0]} content={item[1]} isHTML={!!props.html}/>);
+};
+
+const VersionContent = (props) => {
+  /* Used to render content of Versions.
+  * imageLoadCallback is called to update segment numbers placement
+  * overrideLanguage a string with the language name (full not 2 letter) to force to render to overriding what the content language context says. Can be useful if calling object determines one langugae is missing in a dynamic way
+  * defaultToInterfaceOnBilingual use if you want components not to render all languages in bilingual mode, and default them to what the interface language is
+  * See filterContentTextByLang for more documentation */
+  const langAndContentItems = filterContentTextByLang(props);
+  const [languageToFilter, _] = Sefaria.getContentLang(props.defaultToInterfaceOnBilingual, props.overrideLanguage);
+  function isImage(textChunk) {
+      const pattern = /<img\b[^>]*>/i;
+      return pattern.test(textChunk);
+  }
+  return langAndContentItems.map((item) => {
+      const [lang, content] = item;
+      if (isImage(content)){
+        return(<VersionImageSpan lang={lang} content={content} languageToFilter imageLoadCallback={props.imageLoadCallback}/>);
+      }
+      return (<ContentSpan lang={lang} content={content}  isHTML={true}/>);
+  })
+}
+
+const VersionImageSpan = ({lang, content, languageToFilter, imageLoadCallback}) => {
+    function getImageAttribute(imgTag, attribute) {
+      const parser = new DOMParser();
+      const doc = parser.parseFromString(imgTag, 'text/html');
+      const imgElement = doc.querySelector('img');
+
+      if (imgElement) {
+        return imgElement.getAttribute(attribute);
+      }
+
+      return null;
+    }
+    const altText = getImageAttribute(content, 'alt');
+    const srcText = getImageAttribute(content, 'src');
+    content = (<div className="image-in-text">{<img onLoad={imageLoadCallback} src={srcText} alt={altText}/>}<p className="image-in-text-title">{altText}</p></div>);
+    if (lang === 'he' && languageToFilter === "bilingual") {content = ''}
+
+    return(<span className={`contentSpan ${lang}`} lang={lang} key={lang}>{content}</span>)
+};
+
+const filterContentTextByLang = ({text, html, overrideLanguage, defaultToInterfaceOnBilingual=false, bilingualOrder = null}) => {
```

Comment:
Let's add an underscore at the beginning of this func name to indicate it's for local use